cost/aws/unused_volumes/aws_delete_unused_volumes.pt

```diff
@@ -709,7 +700,7 @@ script "js_volume_cost_mapping", type:"javascript" do
   result={
     "unused_volumes": unused_volumes,
     "message": message,
-    "accountId": ds_get_caller_identity[0]['account']
+    "accountID": String(ds_get_caller_identity[0]['account'])
```

Comment:
Checked with Ren, this `String()` call was not needed in the end, and will be removed.